src/rhsmlib/facts/host_collector.py

```diff
@@ -74,18 +66,6 @@ def get_all(self):
         host_facts.update(virt_collector_info)
         host_facts.update(firmware_info_dict)
 
-        default_rhsm_dir = rhsm.config.DEFAULT_CONFIG_DIR.rstrip('/')
-        custom_facts_dir = os.path.join(default_rhsm_dir, self.facts_sub_dir)
-        path_and_globs = [(custom_facts_dir, self.facts_glob)]
-
-        custom_facts = custom.CustomFactsCollector(
-            prefix=self.prefix,
-            testing=self.testing,
-            path_and_globs=path_and_globs
-        )
-        custom_facts_dict = custom_facts.get_all()
-        host_facts.update(custom_facts_dict)
```

Comment:
@awood was there a reason for having `HostCollector` run `CustomFactsCollector` internally?